.github/workflows/test.yml

```diff
@@ -66,6 +67,8 @@ jobs:
           - { name: mattersim, test_path: "tests/models/test_mattersim.py" }
           - { name: orb, test_path: "tests/models/test_orb.py" }
           - { name: sevenn, test_path: "tests/models/test_sevennet.py" }
+          - { name: fairchem, test_path: "tests/models/test_fairchem.py" }
+          - { name: metatensor, test_path: "tests/models/test_metatensor.py" }
```

Comment:
Can we make these alphabetical? you've duplicated the fairchem tests so maybe merge main then do a soft reset against main and then force pushing might be a useful thing to do just to check that there are no other divergence issues.

Reply:
Yes, of course